server/modlog.ts

```diff
@@ -476,8 +476,8 @@ type ModlogResult = ModlogEntry | undefined;
 export const PM = new QueryProcessManager<ModlogTextQuery, ModlogResult[]>(module, async data => {
 	const {rooms, regexString, maxLines, onlyPunishments} = data;
 	try {
-		const results = await Rooms.Modlog.runTextSearch(rooms, regexString, maxLines, onlyPunishments);
-		return results.map((line: string, index: number) => parseModlog(line, results[index + 1]));
+		const results = await Rooms.Modlog?.runTextSearch(rooms, regexString, maxLines, onlyPunishments);
```

Comment:
I don't like that this change infects all callers, it isn't a nice interface to use. I think `Rooms.Modlog` should always be set and these calls should just be no-ops if you don't have SQLite enabled